domains/renzyx.json

```diff
@@ -6,6 +6,11 @@
     "email": "alex.arias.jorym@gmail.com"
   },
   "record": {
-    "CNAME": "xyugen.github.io"
+    "CNAME": "xyugen.github.io",
+    "MX": "feedback-smtp.us-east-1.amazonses.com",
+    "TXT": [
+      "v=spf1 include:amazonses.com ~all",
+      "p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDl3uimKrfDsLkGKkuwIhxEi23dmRf1GTyrcfpzF/iSfGrMp5xbDo3+lqN4R3Dx48orvlGbtNxQevx4NPIHU+BAIfRyRkMDzP9X+elTMFkHqHtly5OswQJd0Jciax8WqPoTfLo7+cjDyY0lejoqadUK0gxRe5b5HOLGNUBOCgmCNQIDAQAB"
```

Comment:
You can't use CNAME with other records

either remove it or convert it to an A record